Monika After Story/game/script-topics.rpy

```diff
@@ -7254,23 +7254,23 @@ init 5 python:
 
 label monika_solipsism:
     m 3eub "Have you ever heard about solipsism, [player]?"
-    m 3eua "It's a philosophical idea which states that only your mind is sure to exist."
-    m 1eud "What do you think of that?"
-    m "Is there really something other than yourself?"
-    m 1euc "Or maybe you and everything else are just some representations of my psyche?"
-    m 1dsd "Extensions of my own consciousness..."
-    m 1hub "Ahaha!"
-    m 1eub "I'm just kidding, of course."
-    m 1eua "It's more likely that {i}I'm{/i} a representation of your mind than the other way around."
-    m "I'm the 'game character', right?"
-    m 1dsc "Still...it makes you wonder about the nature of our realities. Or maybe 'fabric' would be a better term than 'nature'?"
-    m 1eua "Anyway, I recommend reading more about solipsism."
-    m 1eub "There are several varieties of it."
-    m "Like metaphysical, epistemological, or methodological solipsism."
-    m 1eua "You can read more about them online."
-    m "Then we could talk about these topics more in-depth."
-    m 1hub "I'm really looking forward to talking about philosophy with you, [player]!"
-    m 1eua "I want to know more about the way you think..."
+    m 3eua "It's an idea which states that only you yourself exists."
+    m 1eud "'Cogito, ergo sum.'"
+    m "'I think, therefore I am.'"
+    m "If you're sure your self exists, how about others outside your self?"
+    m 1euc "Maybe everyone else is just a figment of our imagination, [player]."
```

Comment:
```suggestion
    m 3etc "Maybe everyone else is just a figment of our imagination, [player]?"
```

Reply:
Expression is good. Sentence is meant to be a statement rather than a question so the mark is unnecessary.

Reply:
Imo fits better as a question